src/js/components/DataTableColumns/stories/Inline.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import { Data, DataFilters, DataTable, Grid } from 'grommet';
+
+import { DataTableColumns } from '../DataTableColumns';
+import { columns, DATA } from '../../DataTable/stories/data';
+
+export const Inline = () => (
+  // Uncomment <Grommet> lines when using outside of storybook
+  // <Grommet theme={...}>
+  <Grid pad="large" columns={['large']} justifyContent="center">
+    <Data data={DATA} updateOn="change">
+      <DataFilters>
+        <DataTableColumns
+          options={columns.map(({ header, property }) => ({
+            property,
+            label: header,
```

Comment:
<img width="806" alt="Screen Shot 2023-01-03 at 12 42 21 PM" src="https://user-images.githubusercontent.com/54560994/210429562-512216b7-10c3-4190-bf7e-dcd9ce31ad38.png">

In the grommet theme the bolding on 'Location' and 'Date' doesn't match up with the table headers like it does in the HPE theme

<img width="801" alt="Screen Shot 2023-01-03 at 12 43 03 PM" src="https://user-images.githubusercontent.com/54560994/210429690-ea05c3f4-a83e-48fa-a008-4c430a632572.png">

In addition when the label is not bolded it's associated down/up buttons are not read correctly by the screenreader.

Bolded label (Location) up button screenreader output: '1 Location move up button'
Non bolded label (Name with extra) up button screenreader output: '2 [object, object] move up button'